src/playground/with-layout/select/select-clean.component.html

```diff
@@ -1,5 +1,6 @@
 <nb-select placeholder="Cleanable">
   <nb-option>Clean</nb-option>
+  <nb-option [value]="nullValue">Clean with null value</nb-option>
```

Comment:
Why do you use property set to null instead of setting null in input directly?
`[value]="null"`
Also, what do you think about adding one more option without value, to illustrate another reset way to make reset option.